documentation/modules/rn-2.6.adoc

```diff
@@ -309,6 +316,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Migrating RHEL 7.0 and 7.1 from VMware 7 to {ocp} 4.13, the name of the network interfaces changes
```

Comment:
`.Migrating RHEL 7.0 and 7.1 from VMware 7 to {ocp} 4.13, the name of the network interfaces changes`

to
`.Migrating RHEL and CentOS 7.0 and 7.1 from VMware to {ocp}, the name of the network interfaces changes`